static/src/js/components/GranuleResults/__tests__/GranuleResultsItem.test.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable testing-library/no-unnecessary-act */
```

Comment:
Is this covered by eslintrc? If not, use `// eslint-disable-next-line` instead of disabling for the whole file